AccessibilityInsightsForAndroidService/app/src/main/AndroidManifest.xml

```diff
@@ -58,5 +58,10 @@
             android:exported="true"
             android:permission="android.permission.SET_DEBUG_APP">
         </provider>
+        <provider
+            android:name="androidx.work.impl.WorkManagerInitializer"
+            android:authorities="${applicationId}.workmanager-init"
```

Comment:
Is the applicationId variable generated by the WorkManagerHolder?

Reply:
It's defined at https://github.com/microsoft/accessibility-insights-for-android-service/blob/main/AccessibilityInsightsForAndroidService/app/build.gradle#L9:

```
 applicationId "com.microsoft.accessibilityinsightsforandroidservice"
```

The default build process then replaces the `${applicationId}` variable with the value from the gradle definition.